ocs_ci/helpers/tlsprofile_helper.py

```
@@ -351,6 +466,28 @@ def _tls_scan_discover_endpoints(kubeconfig, namespaces, component="all"):
+                            "process": process,
+                        }
+                    )
+            if extra_template:
+                for extra in extra_ports:
+                    if extra not in declared_ports:
+                        endpoints.append(
+                            {
+                                **extra_template,
+                                "port": str(extra),
+                            }
+                        )
```

Ruff (0.16.2)

[warning] Use `list.extend` to create a transformed list

(PERF401)